src/components/sortControl/sortControl.tsx

```diff
@@ -0,0 +1,34 @@
+import './sortControl.css';
+
+interface SortControlProps {
+  sortOptions: movieSortOption[];
+  defaultSelection: movieSortOption;
+  onSelect: (selectedOption: movieSortOption) => void;
+}
+
+const SortControl = (props: SortControlProps) => {
+  const onChangeHandler = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    console.log('Hi');
```

Comment:
Please configure no-console lint rule to prevent consoles in PR (https://eslint.org/docs/latest/rules/no-console)